File: exercise-2/getSandwichInfo.js
function getSandwichInfo() {
  var sandwiches = ['Philly cheesesteak', 'Italian sub', 'Reuben', 'Bahn Mi', 'Nashville Hot Chicken', 'Caprese']
  var whatIsShift = sandwiches.shift()
  var whatIsPop = sandwiches.pop()
  var whatIsLength = sandwiches.length
  var whatIsIndex = sandwiches[2]

  for (var i = sandwiches.length - 1; i >= 0; i--) {
    console.log(`Hmmmm... Lemme get uh... Ummm... I'll take a ${sandwiches[i]} please. Thanks.`)
  }

  var dinnerSpecial = [sandwiches[1], sandwiches[sandwiches.length - 1], 'fries', 'two drinks']
  console.log(dinnerSpecial[0])
  dinnerSpecial[3] = 'a coupla Baja Blasts'
  console.log(dinnerSpecial[2])
  dinnerSpecial[2] = 'onion rings'

  var dailySpecial = [sandwiches[0], 'chips', 'drink']
  console.log(dailySpecial[2])
  dailySpecial[0] = 'Chesseburkle'
  console.log(dailySpecial)
  dailySpecial[1] = sandwiches[3]
}

File: exercise-2/get_sandwich_info.rb
def get_sandwich_info() 
  sandwiches = ['Philly cheesesteak', 'Italian sub', 'Reuben', 'Bahn Mi', 'Nashville Hot Chicken', 'Caprese']
  what_is_shift = sandwiches.shift()
  what_is_pop = sandwiches.pop()
  what_is_length = sandwiches.length
  what_is_index = sandwiches[2]

  sandwiches.each do |sandwich|
    puts "Hmmmm... Lemme get uh... Ummm... I'll take a #{sandwich} please. Thanks."
  end

  dinner_special = [sandwiches[1], sandwiches[sandwiches.length - 1], 'fries', 'two drinks']
  puts dinner_special[0]
  dinner_special[3] = 'a coupla Baja Blasts'
  puts dinner_special[2]
  dinner_special[2] = 'onion rings'

  daily_special = [sandwiches[0], 'chips', 'drink']
  puts daily_special[2]
  daily_special[0] = 'Chesseburkle'
  puts daily_special
  daily_special[1] = sandwiches[3]
end